libkbfs/folder_block_manager.go

```diff
@@ -1306,6 +1308,74 @@ func (fbm *folderBlockManager) clearLastQRData() {
 	fbm.lastReclamationTime = time.Time{}
 }
 
+func (fbm *folderBlockManager) doChunkedGetNonLiveBlocks(
+	ctx context.Context, ptrs []BlockPointer) (
+	nonLiveBlocks []kbfsblock.ID, err error) {
+	fbm.log.CDebugf(ctx, "Get live count for %d pointers", len(ptrs))
+	bops := fbm.config.BlockOps()
+
+	// Round up to find the number of chunks.
+	numChunks := (len(ptrs) + numPointersToDowngradePerChunk - 1) /
+		numPointersToDowngradePerChunk
+	numWorkers := numChunks
+	if numWorkers > maxParallelBlockPuts {
+		numWorkers = maxParallelBlockPuts
```

Comment:
Shouldn't this be its own constant? It doesn't really have anything to do with block put parallelism.